src/transform.ts

```diff
@@ -233,7 +233,7 @@ export function isImputeSequence(t: ImputeSequence | any[] | undefined): t is Im
   return t && t['start'] !== undefined && t['stop'] !== undefined;
 }
 
-export interface ImputeTransform {
+export interface ImputeTransform extends ImputeBase {
```

Comment:
This is wrong.  ImputeTransform doesn't have `frame`.

Reply:
Oh I see -- you actually want VL impute transform to support more than Vega.  That's a good idea.